pkg/reconciler/knativeserving/ingress/kourier_test.go

```diff
@@ -158,6 +180,16 @@ func verifyGatewayServiceType(t *testing.T, u *unstructured.Unstructured, expSer
 	}
 }
 
+func verifyGatewayServiceLoadBalancerIP(t *testing.T, u *unstructured.Unstructured, expServiceLoadBalancerIP string) {
+	if u.GetKind() == "Service" && u.GetName() == kourierGatewayServiceName {
+		svc := &v1.Service{}
+		err := scheme.Scheme.Convert(u, svc, nil)
+		util.AssertEqual(t, err, nil)
+		svcLoadBalancerIP := svc.Spec.LoadBalancerIP
+		util.AssertDeepEqual(t, string(svcLoadBalancerIP), expServiceLoadBalancerIP)
```

Comment:
```suggestion
		util.AssertDeepEqual(t, svcLoadBalancerIP, expServiceLoadBalancerIP)
```

Could you fix the lint error?